### DISCUSSION TITLE
vitepress使用build打包和preview启动时,还支持md的热更新吗

### DISCUSSION THREAD

**Discussion by LLLPY** [vuejs/vitepress#3421] [Q&A]
vitepress使用build打包和preview启动时,还支持md的热更新吗

**Comment by brc-dd** [comment]
No

**Reply by LLLPY** [reply]
OK

**Comment by LLLPY** [comment]
如何通过cdn引入第三方库，比如引入element-plus（在不build的情况下）

**Reply by brc-dd** [reply]
Add them using head https://vitepress.dev/reference/site-config#head

**Reply by LLLPY** [reply]
i know this way, but i can only add css file in this way, if i need to use a component, i have to register it here.i can't register if i use head🫠
<img width="1512" alt="image" src="https://github.com/vuejs/vitepress/assets/61370857/4fbec4b4-2b4c-4e82-80b6-1d5ece1eef86">

**Reply by brc-dd** [reply]
Yeah, it's because ElementPlus' CDN version is meant to be used with CDN version of vue but vitepress bundles it. If manually listing everything in theme/index.js is an issue, you can use on demand imports - https://element-plus.org/en-US/guide/quickstart.html#on-demand-import

**Reply by LLLPY** [reply]
it seems didn't work...i setup in dev mode. is this only useful in build?
<img width="1512" alt="image" src="https://github.com/vuejs/vitepress/assets/61370857/83724921-6ec3-4519-b6cf-6c370f373b4e">
<img width="1512" alt="image" src="https://github.com/vuejs/vitepress/assets/61370857/674d16c2-2e65-43a5-a912-ad73db62d649">

**Reply by brc-dd** [reply]
The CDN build of vue won’t work with VitePress. You need to install element plus and use it. If you want to auto import stuff you can configure vite plugins, but CDN version of element-plus or vue won’t work.
